packages/features/ee/dsync/lib/handleGroupEvents.ts

```diff
@@ -190,6 +191,15 @@ const handleGroupEvents = async (event: DirectorySyncEvent, organizationId: numb
         });
       }),
     ]);
+    // Add users to team event types if assignAllTeamMembers is enabled
```

Comment:
Adding this after the other `promise.all` since it creates a user's profile and we want to make sure that succeeds first.